src/status_im/ui/components/emoji_thumbnail/styles.cljs

```diff
@@ -0,0 +1,179 @@
+(ns status-im.ui.components.emoji-thumbnail.styles
+  (:require [quo.design-system.colors :as colors]
+            [status-im.ui.components.emoji-thumbnail.utils :as emoji-utils]))
+
+(defn emoji-thumbnail-icon [color size]
+  {:width            size
+   :height           size
+   :align-items      :center
+   :justify-content  :center
+   :border-radius    (/ size 2)
+   :background-color color
+   :border-width     0.5
+   :border-color     "rgba(0,0,0,0.1)"})
+
+(defn emoji-thumbnail-icon-text [size]
+  {:font-size   (emoji-utils/emoji-font-size size)
+   :line-height size
+   :margin-top  (emoji-utils/emoji-top-margin-for-vertical-alignment size)})  ;; Required for vertical alignment bug - Check function defination for more info
+
+
+;; Styles Related to Emoji Thumbnail Picker
+
+(def emoji-thumbnail-preview-size 60)
+
+(def emoji-picker-upper-components-size 350)
+
+(defn emoji-picker-gray-color []
+  (if (colors/dark?) "#c3c3bc99" "#3C3C4399"))
+
+(defn emoji-picker-category-or-search-container []
+  (if (colors/dark?) "#110d0a" "#EEF2F5"))
+
+(defn emoji-picker-active-category-container []
+  (if (colors/dark?) "#87877f33" "#78788033"))
+
+(defn emoji-picker-active-category-color []
+  (if (colors/dark?) "#bbbbbb" "#000000"))
+
+(def emoji-thumbnail-preview
+  {:margin-top       16
+   :margin-bottom    16
+   :align-items      :center
+   :justify-content  :center})
+
+(defn emoji-picker-keyboard-container []
+  {:border-radius    0
+   :background-color (colors/get-color :ui-background)})
+
+(defn emoji-picker-search-bar []
+  {:border-radius    10
+   :height           36
+   :background-color (emoji-picker-category-or-search-container)})
+
+(defn emoji-picker-search-bar-text []
+  {:color (emoji-picker-gray-color)})
+
+(defn emoji-picker-header []
+  {:font-size     13
+   :color         (emoji-picker-gray-color)
+   :font-weight   "600"
+   :margin-top    7
+   :margin-bottom 0})
+
+(defn emoji-keyboard [func]
+  {:onEmojiSelected              func
+   :emojiSize                    23
+   :containerStyles              (emoji-picker-keyboard-container)
+   :categoryPosition             "floating"
+   :categoryContainerColor       (emoji-picker-category-or-search-container)
+   :activeCategoryColor          (emoji-picker-active-category-color)
+   :activeCategoryContainerColor (emoji-picker-active-category-container)
+   :categoryColor                (emoji-picker-gray-color)
+   :enableSearchBar              true
+   :searchBarTextStyles          (emoji-picker-search-bar-text)
+   :searchBarStyles              (emoji-picker-search-bar)
+   :searchBarPlaceholderColor    (emoji-picker-gray-color)
+   :closeSearchColor             (emoji-picker-gray-color)
+   :headerStyles                 (emoji-picker-header)
+   :disabledCategory             ["symbols"]})
+
+(def emoji-picker-colors-row1
```

Comment:
these don't look like styles, they have data, so probably better to move them to component ns